plugins/web/opentelemetry-instrumentation-user-interaction/src/instrumentation.ts

```diff
@@ -680,7 +680,7 @@ export class UserInteractionInstrumentation extends InstrumentationBase<UserInte
   /**
    * returns Zone
    */
-  getZoneWithPrototype(): ZoneTypeWithPrototype | undefined {
+  private _getZoneWithPrototype(): ZoneTypeWithPrototype | undefined {
```

Comment:
this is the breaking change - IIUC this was not intended to be exposed to the end-user. `ZoneWithPrototype` is in from `internal-types` and it leaked Zone.js types to the public API.